sources/south-america/br/Rio_2013.geojson

```diff
@@ -7,13 +7,611 @@
         "country_code": "BR",
         "name": "Rio Mosaic 2013",
         "available_projections": [
+<<<<<<< HEAD
```

Comment:
I have no idea where that conflict comes from, the `Rio*` files was not edited, to what I know, only source of conflict I can think of is not syncing the repository before creating new files, which shouldn't be causing conflicts at all.